<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-JEI8AS0O" ignored="0" name="Test" tests="2" failures="1" timestamp="2022-10-16T15:49:14 EDT" time="69.784" errors="0">
  <testcase name="LoginSuccess_001" time="5.213" classname="testcases.LoginTest"/>
  <testcase name="LoginUnsuccessful_002" time="47.595" classname="testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [Invalid Credentials] but found [Invalid credentials]">
      <![CDATA[java.lang.AssertionError: expected [Invalid Credentials] but found [Invalid credentials]
at testcases.LoginTest.LoginUnsuccessful_002(LoginTest.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LoginUnsuccessful_002 -->
</testsuite> <!-- Test -->
